lib/matplotlib/tests/test__macosx.py

```diff
@@ -0,0 +1,34 @@
+import sys
+import pytest
+from matplotlib import cbook
+
+
+class NoString:
+    def __repr__(self):
+        pass
+
+
+if sys.platform == "darwin":
```

Comment:
can you use `pytest.skipif` instead?

Reply:
Sure! Might be a bit before I can make a commit.